### PR TITLE
Merge Pro_LoadTestV26 into catalyst

### PR DESCRIPTION
**Pull request by emranemran**
Bring in latest load test features/updates into catalyst.

### CODE REVIEW

**Comment by emranemran** [comment]
The only merge conflicts were a few minor ones in CMakeLists.txt

File: CMakeLists.txt [hunk 1]
```diff
@@ -960,5 +960,5 @@ add_executable(streamstatustest test/status.cpp)
 target_link_libraries(streamstatustest mist)
 add_executable(websockettest test/websocket.cpp)
 target_link_libraries(websockettest mist)
-add_executable(MistLoadTest test/load.cpp ${BINARY_DIR}/mist/.headers generated/loadtest_html.h)
```
Comment (emranemran) [inline CMakeLists.txt]: FYI: removed this unused reference to .headers to fix build errors.

**Review by iameli — APPROVED** [review]
Cool, looks good assuming CI passes

File: CMakeLists.txt [hunk 2]
```diff
@@ -344,6 +344,10 @@ if (NOT NOSSL)
   find_path(SRTP2_HEADER_DIR "srtp2/srtp.h")
   include_directories(${MBEDTLS_HEADER_DIR} ${SRTP2_HEADER_DIR})
 endif()
+if(SRT_LIB)
+  target_sources(mist PRIVATE lib/socket_srt.h PRIVATE lib/socket_srt.cpp)
+  target_link_libraries(mist srt)
+endif()
```
[the remark this answers is not in the document]
Comment (emranemran) [reply CMakeLists.txt]: I'll revisit and try to solve this libsrt dependency correctly after merging this to unblock.

File: lib/dtsc.cpp [hunk 3]
```diff
@@ -2198,7 +2198,7 @@ namespace DTSC{
   void Meta::removeTrack(size_t trackIdx){
     if (!getValidTracks().count(trackIdx)){return;}
     Track &t = tracks[trackIdx];
-    for (uint64_t i = t.pages.getDeleted(); i < t.pages.getEndPos(); i++){
+    for (size_t i = t.pages.getDeleted(); i < t.pages.getEndPos(); i++){
```
[the remark this answers is not in the document]
Comment (emranemran) [reply lib/dtsc.cpp:2201]: Do you mean break at compile time or run time?

[the remark this answers is not in the document]
Comment (iameli) [reply lib/dtsc.cpp:2201]: Compile. Looks like this builds tho

**Pull request merged**